File: app/src/main/java/com/example/thomasskogemann/flashcard/adapters/HeaderViewHolder.java
package com.example.thomasskogemann.flashcard.adapters;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.example.thomasskogemann.flashcard.R;

import butterknife.ButterKnife;

/**
 * Created by Thomas Skogemann on 28-09-2016.
 */
public class HeaderViewHolder extends RecyclerView.ViewHolder {
    TextView headerTextView;

    public HeaderViewHolder(View itemView) {
        super(itemView);
        headerTextView = ButterKnife.findById(itemView, R.id.questionHeaderView);
    }
}

File: app/src/main/java/com/example/thomasskogemann/flashcard/adapters/HeaderViewHolder.kt
package com.example.thomasskogemann.flashcard.adapters

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView

import com.example.thomasskogemann.flashcard.R

import butterknife.ButterKnife

/**
 * Created by Thomas Skogemann on 28-09-2016.
 */
class HeaderViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    internal var headerTextView: TextView

    init {
        headerTextView = ButterKnife.findById<TextView>(itemView, R.id.questionHeaderView)
    }
}
